boring-sys/cmake/arm-linux.cmake

```diff
@@ -0,0 +1,3 @@
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR armv7)
```

Comment:
This is probably okay for now, but if someone ever wants to use the `arm-unknown-linux-gnueabihf` target to compile for ARMv6 Linux we'll have to change it, so let's rename the config file "armv7-linux" to be explicit.

Reply:
Will do. Probably change the target to armv7 as well in `build.rs`.

Reply:
I think that one's correct as is; it looks like [`CARGO_CFG_TARGET_ARCH`][https://doc.rust-lang.org/reference/conditional-compilation.html#target_arch] is different from "the architecture part of `TARGET`". We could change what the code is sniffing, but that's a bigger change.